apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/SyncOnboardingCompanion.tsx

```diff
@@ -479,12 +481,25 @@ const SyncOnboardingCompanion: React.FC<SyncOnboardingCompanionProps> = ({
     };
   }, [device, allowedError, handleDesyncTimerRunsOut, desyncTimeout]);
 
+  const recoverServices = useFeature("protectServicesDesktop");
+  const recoverStoreId = recoverServices?.params?.protectId ?? "";
+
+  const [recoverOnboardingStatus, setRecoverOnboardingStatus] = useState<boolean>();
+
+  const getRecoverOnboardingStatus = useCallback(() => {
+    const userId = getUserId();
+    const status = getStoreValue(`USER_RESTORE_ONBOARDING_STATUS-${userId}`, recoverStoreId);
+    setRecoverOnboardingStatus(status === "true");
+  }, [recoverStoreId]);
+
```

Comment:
as you use it twice the same way, and one other time in more concise way.

It could be great to make this as a hook and use it in the 3 files maybe ? 
Wdyt? :)